pkg/admissioncontroller/webhook/admission/seedrestriction/handler.go

```diff
@@ -71,24 +72,26 @@ func (h *Handler) Handle(ctx context.Context, request admission.Request) admissi
 
 	requestResource := schema.GroupResource{Group: request.Resource.Group, Resource: request.Resource.Resource}
 	switch requestResource {
-	case backupBucketResource:
-		return h.admitBackupBucket(ctx, seedName, request)
 	case backupEntryResource:
 		return h.admitBackupEntry(ctx, seedName, request)
+	case backupBucketResource:
+		return h.admitBackupBucket(ctx, seedName, request)
```

Comment:
Is this automatically generated code or why was this section moved? With regards to alphabetic ordering it was better before.
WDYT?

Reply:
Sorry, reverted 😄